services/net/clip/ClipAction.cs

```diff
@@ -385,26 +385,23 @@ private static async Task<long> GetDurationAsync(string inputFile)
         switch (ext)
```

Comment:
Move logic to `ParseDurationAsync(...)` function.  As it's specifically related to parsing the duration.